src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicSpawnerTag.java

```diff
@@ -0,0 +1,334 @@
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.mobs>
+        // @returns ListTag(EntityTag)
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Returns a ListTag of all active MythicMobs from this spawner.
+        // -->
+        registerTag("mobs", (attribute, object) -> {
+            ArrayList<EntityTag> list = new ArrayList<>();
+            for (UUID uuid : object.getSpawner().getAssociatedMobs()) {
+                list.add(new EntityTag(MythicMobsMobTag.valueOf(uuid.toString()).getEntity()));
+            }
+            return new ListTag(list);
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.location>
+        // @returns LocationTag
+        // @plugin Depenizen, MythicMobs
+        // @mechanism MythicSpawnerTag.location
+        // @description
+        // Returns a LocationTag of the MythicSpawner's location.
+        // -->
+        registerTag("location", (attribute, object) -> {
+            AbstractLocation loc = object.getSpawner().getLocation();
+            return new LocationTag((World) loc.getWorld(), loc.getX(), loc.getY(), loc.getZ());
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.group>
+        // @returns ElementTag
+        // @plugin Depenizen, MythicMobs
+        // @mechanism MythicSpawnerTag.group
+        // @description
+        // Returns the MythicSpawner's group, if applicable.
+        // -->
+        registerTag("group", (attribute, object) -> {
+            return new ElementTag(object.getSpawner().getGroup());
+        });
+    }
+
+    @Override
+    public void adjust(Mechanism mechanism) {
+
+        // <--[mechanism]
+        // @object MythicSpawnerTag
+        // @name group
+        // @input ElementTag
+        // @description
+        // Sets the group of the MythicSpawner.
+        // @tags
+        // <MythicSpawnerTag.group>
+        // -->
+        if (mechanism.matches("group")) {
+            spawner.setGroup(mechanism.getValue().asString());
+        }
+
+        // <--[mechanism]
+        // @object MythicSpawnerTag
+        // @name location
+        // @input LocationTag
+        // @description
+        // Sets the location of the MythicSpawner.
+        // @tags
+        // <MythicSpawnerTag.location>
+        // -->
+        else if (mechanism.matches("location") && mechanism.requireObject(LocationTag.class)) {
+            spawner.setLocation(BukkitAdapter.adapt(mechanism.valueAsType(LocationTag.class)));
+        }
+
+        // <--[mechanism]
+        // @object MythicSpawnerTag
+        // @name cooldown
+        // @input ElementTag(Number)
+        // @description
+        // Sets the MythicSpawner's cooldown timer, in seconds.
+        // @tags
+        // <MythicSpawnerTag.cooldown>
+        // -->
+        else if (mechanism.matches("cooldown") && mechanism.requireInteger()) {
+            spawner.setCooldownSeconds(mechanism.getValue().asInt());
```

Comment:
This should take a DurationTag as input

Reply:
I didn't do a DurationTag, due to not supporting ticks.

Reply:
The relevant tag is a DurationTag... the tag's output should work as input to the mech.